src/vcModals.cpp

```diff
@@ -470,7 +470,9 @@ void vcModals_DrawUnsupportedFiles(vcState *pProgramState)
       ImGui::NextColumn();
 
       ImGui::GetCurrentWindow()->DC.CurrentLineTextBaseOffset = offset;
-      ImGui::TextUnformatted(udResultAsString(pProgramState->errorItems[i].resultCode));
+
+      const char *errorStr = udTempStr("error%s", udResultAsString(pProgramState->errorItems[i].resultCode));
```

Comment:
We typically prefix string types with `p`. Additionally we don't typically shorten words. So in this case this variable name should be `const char *pErrorString = ...`.

Reply:
Got it. Thanks.

Reply:
This will also cause most errors to not be translated- I would recommend a switch of the specifically handled error with a `default` for generic errors

Reply:
The udResultAsString is used only in the unsupported files window currently, I translated those errors related with files for this quick fix. I think if we don't use it elsewhere, that's fine, otherwise we need to create a new interface.

Reply:
And there is another issue, we show "open failure" for a empty folder, it's a litter strange. A default message can resolve this problem, I'll talk to our QA about this fix.

Reply:
Fixed both variable name and a default string for generic errors.